marin/post_training/allocate_tpu.sh

```diff
@@ -0,0 +1,32 @@
+TPU_NAME="post-training-v5p-8"
```

Comment:
Can you submit the PR against @sherryy 's branch so it's easier to see the diff?  (Or I guess wait until she's merged.)